### PR TITLE
Map modal fix

### PR DESCRIPTION
**Pull request by valeriegc**
Fixed overall CSS in the map modal and adjusted the on hover properties of the buttons

### CODE REVIEW

File: src/lib/ToastMessage.svelte [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<p>Map copied</p>
```
Comment (AlexanderKolberg) [inline src/lib/ToastMessage.svelte]: Since this is not a reusable component, should it just be in the modal?
That way you can avoid the store too

Comment (valeriegc) [reply src/lib/ToastMessage.svelte]: done

Comment (valeriegc) [reply src/lib/ToastMessage.svelte]: fixed

**Review by AlexanderKolberg — APPROVED** [review]
LGTM

**Pull request merged**